README.md

```diff
@@ -1,4 +1,4 @@
-ACL (Access Control List)
+acl
```

Comment:
I know it's down in the TOC, but it may be better to happen as soon as possible.